coil-default/src/main/java/coil/ImageLoaderProvider.kt

```diff
@@ -0,0 +1,17 @@
+package coil
+
+import android.app.Application
+
+/**
+ * A class that can provide the default [ImageLoader].
+ *
+ * Either implement [ImageLoaderProvider] on your [Application] **or** call [Coil.setImageLoader]
+ * with your [ImageLoaderProvider] to provide the default [ImageLoader].
+ */
+interface ImageLoaderProvider {
+
+    /**
+     * Return the default [ImageLoader].
+     */
+    fun getImageLoader(): ImageLoader
```

Comment:
I see that the CameraX and WorkManager implementations use this `get` syntax. Personally, I would like `create` better than `get`, but for the sake of consistency with other libraries, 👍

Reply:
Yeah, `get` doesn't feel 100% right in this instance, however I'd rather be consistent with AndroidX. Also this method doesn't necessarily have to create a new instance each time it's called.